src/styles/index.scss

```diff
@@ -1,3 +1,13 @@
+@import './utils/variables';
+@import './blocks/card';
+@import './blocks/starts';
```

Comment:
There seems to be a typo in the file name. It's supposed to be 'stars' instead of 'starts'.